controller/client/client.go

```diff
@@ -351,6 +435,38 @@ func (c *Client) StreamJobEvents(appID string, output chan *ct.JobEvent) (stream
 	return c.ResumingStream("GET", fmt.Sprintf("/apps/%s/jobs", appID), output)
 }
 
+func (c *Client) WatchJobEvents(appID string) (*JobWatcher, error) {
+	events := make(chan *ct.JobEvent)
+
+	stream, err := c.StreamJobEvents(appID, events)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return newJobWatcher(events, stream), nil
+}
+
+func (c *Client) ExpectedScalingEvents(actual, expected map[string]int, clusterSize int) ct.JobEvents {
+	events := make(ct.JobEvents)
+	for typ, count := range expected {
+		diff := count
+		val, ok := actual[typ]
+		if ok {
+			diff = count - val
+		}
+		if typ == "omni" {
+			diff *= clusterSize
+		}
```

Comment:
This check is incorrect, the type being named "omni" does not indicate the type is an omni job (it just so happens that this is the case in the tests), the correct check is against the [process type's Omni field](https://github.com/flynn/flynn/blob/master/controller/types/types.go#L49).

Note this handling of omni jobs will most likely be simplified in the next implementation of the scheduler, see #1484.